services/agora/src/api/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-7.12.0
+7.7.0
```

Comment:
Hmm maybe keep the same version? It seems the old version is changing lots of files 
We should probably enforce a version of that script